### ISSUE TITLE
Create Story on telegram desktop

### ISSUE THREAD

**Issue by mr-459** [telegramdesktop/tdesktop#26632]
### Is your feature request related to a problem?

You can only create stories on android telegram for now and its annoying for pc-users like me

### Describe the solution you'd like

a button like the one in android or maybe a button below the "New Group" for "Add new story"
we don't necessarily need to record a video, we mostly just upload fun things as story 

### Describe alternatives you've considered

there is no alternative, i will never use Instagram because it doesn't support desktop well

**Comment by ilya-fedin** [comment]
I bet this depends on #781.

**Comment by john-preston** [comment]
It's not planned, at least for now. But we may leave this as a feature request here.

**Comment by nyakze** [comment]
> I bet this depends on #781.

Not necessarily. It's possible to create stories from previously made photos and videos. Maybe this could be added to desktop client.

**Comment by nahomjc** [comment]
i can access story on  telegram desktop why is that can any body tell me

**Comment by adrianratajczak** [comment]
Definitely this feature should appear in the desktop version of telegram. Right now I create graphics on my computer and have to transfer them to my phone to add Story.

**Comment by VadimVadimVadim1337** [comment]
+1

**Comment by AndreySV** [comment]
Mobile Linux distributions (like Mobian, PostmarketOS and others) use telegram desktop a mobile telegram client. For someone this feature could be useful in telegram desktop.

**Comment by AAraKKe** [comment]
With the update adding stories for channels, not allowing to use them on desktop is unexpected. People who have channels for reaching their audience will very likely work on the stories they publish on other social media on the computer and not mobile phones. Being able to send stories only from the phone creates a barrier for those entities to use Telegram stories to reach their Telegram users that does not exist on other platforms. It seems to me that this is a priority if the intention is for stories to be used.

**Comment by GitHubAssAssIn** [comment]
is this like to avoid automation and stuff

**Comment by ChrisTVH** [comment]
The truth if it is shocking not to have the stories in the desktop version of telegram, is because I also use almost nothing from mobile, and I find it annoying to have to go through the mobile app to make a story.

**Comment by Z4urce** [comment]
+1

**Comment by zahin-zawad** [comment]
+1

**Comment by ivo-ns** [comment]
+1

**Comment by Jaccuse** [comment]
+1

**Comment by muhammadmp97** [comment]
+1

**Comment by stancolend** [comment]
+1

**Comment by zinetx** [comment]
+1 (writing this so it doesn't appear in "similar comments")

**Comment by neifion-00000000** [comment]
uh yeah why is this not a planned feature?  Why is the phone app and desktop app different in features at all for that matter?

**Comment by foodornt** [comment]
JOHN, MOVE YOUR LAZY ASS AND ADD THE STORIES FEATURE TO DESKTOP MY DUDE

**Comment by foodornt** [comment]
lmao we're gonna CONTACT YOUR MANAGER JOHN
THE KARENS ARE COMING

**Comment by Drino955** [comment]
yeeaah, i want to post Linus Torvalds meme stories from my desktop client, but i can't, WTF telegram

**Comment by Linerd4** [comment]
It would be really useful to post photos on Telegram stories directly by desktop. This would be a little nice addition.

**Comment by hamidtech** [comment]
+1

**Comment by girs1982** [comment]
need on desktop

**Comment by akkifireborker** [comment]
If you can upload Shorts to YouTube from desktop and TikToks to... TikTok from desktop, you should be able to upload stories to Telegram from Desktop. There is no reason for it to be any more difficult than just uploading a photo or video to a chat.

**Comment by buzzkirill** [comment]
This would be a super useful feature. It's obviously more convenient to post certain types of stories on the PC.

**Comment by NaRcot01** [comment]
1year later and still no update for that😢

**Comment by kostmetallist** [comment]
+1, also requesting this feature to be prioritized

**Comment by ZDyy1** [comment]
![Uploading 5EB213C6-CC18-4B84-BB93-F3B7A862B5AE.png…]()
still waiting

**Comment by ZDyy1** [comment]
Still waiting -_-

**Comment by LEG1ON-Group** [comment]
+1

**Comment by vehosah** [comment]
@john-preston how many years we need to wait for you to move your ass 😂

**Comment by adrianratajczak** [comment]
> [@john-preston](https://github.com/john-preston) how many years we need to wait for you to move your ass 😂

2 years of notifications. 
Maybe I will just find a best way to run android version on Windows...

**Comment by vehosah** [comment]
> It's not planned, at least for now. But we may leave this as a feature request here.

Durov should have fired you lmao 😂

**Comment by vehosah** [comment]
> > [@john-preston](https://github.com/john-preston) how many years we need to wait for you to move your ass 😂
> 
> 2 years of notifications. Maybe I will just find a best way to run android version on Windows...

😂

**Comment by Hi-Angel** [comment]
⁺¹, posting a story from mobile is convoluted when you have edited the pictures on your desktop.

**Comment by kalecream** [comment]
+1. I can view stories on desktop, thankfully, but I shouldn't need access to a smartphone to post my own stories.

**Comment by LEG1ON-Group** [comment]
> Just to be clear, we're paying customers here, the feature is not available to non-paying users anyway. So it is odd to see it gets no priority.

well, I'm a telegram premium user, when will you add this feature? ;D

**Comment by Den7me** [comment]
It feels like Telegram is half-done / crippled and missies out on core features such as having an option to post stories from desktop clients. In my humble opinion that's not the feature that has to be available to paying customers only.

**Comment by dietomba** [comment]
It's really a shame that this feature has not yet been implemented, considering that one of the strengths that has always distinguished Telegram is an excellent cross-platform experience. But perhaps cryptocurrencies are now more important 😟

**Comment by Den7me** [comment]
> It's really a shame that this feature has not yet been implemented, considering that one of the strengths that has always distinguished Telegram is an excellent cross-platform experience. But perhaps cryptocurrencies are now more important 😟

Well, it is implemented for desktop Premium (paying) users only. Non-paying desktop users cannot use it. Although it is available on their app in all flavors. But the app has a "mystery" as some free accounts can easily post stories without any issues at all and some other free accounts say it requires upgrading to Premium to post stories. It totally beats me what it depends on.

**Comment by Hi-Angel** [comment]
> Well, it is implemented for desktop Premium (paying) users only

Huh? Since when? Can you show a screenshot?

FTR, premiumness is beside the point here, because posts are a premium-only feature in general.

**Comment by Den7me** [comment]
Somebody's said it here before that as a paying customer they have this desktop fearure available so sak them. I am not a premimium member, but I can post stories over the app without an issue.

-----Original Message-----

**Comment by HirushanBandara** [comment]
**This has not been resolved after two years💔**

**Comment by genes2160** [comment]
> **This has not been resolved after two years💔**

its weird that after 2-3 years this feature request hasnt been done yet.... Meanwhile whatapp made it available on whatsapp web.

**Comment by TDYQ-Liu** [comment]
Now is 2026 ☹️.

**Comment by LEG1ON-Group** [comment]
im from 2050. Durov officially announced that this feature will not be added at all
(btw ton is scam too, dont buy it)

**Comment by Kanezal** [comment]
omg, one of the biggest messenger on a PLANET, and yet to stories edit on PC client. wtf
